### ISSUE TITLE
Missing references in plugin

### ISSUE THREAD

**Issue by RFC1920** [Facepunch/Rust.ModLoader#6]
After writing in a few prefixes and a transpiler to modify some game functions for Hurt(), etc., I decided that some of what I was trying to do needed to be moved into an actual plugin.  So, my plugin contains only non-harmony things.

However, I am getting a ton of errors, and my plugin will not compile.  Among other things:
(56,50): error CS0122: 'Vector3' is inaccessible due to its protection level
(71,69): error CS0234: The type or namespace name 'Timers' does not exist in the namespace 'System' (are you missing an assembly reference?)
(81,56): error CS0012: The type 'FacepunchBehaviour' is defined in an assembly that is not referenced. You must add a reference to assembly 'Facepunch.UnityEngine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.

Basically, anything I am trying to use from the available libraries is unavailable.  I tried copying some or all of the dlls from the game into HarmonyMods, but that has had no impact.  I'm just not clear that I am using the right namespace, how to reference correctly, etc.

Currently, my plugin starts with one of the two, both with the same issue:

namespace Rust.ModLoader
namespace Rust.ModLoader.Scripting

FWIW, I have been working on and writing oxide plugins for many years, so I have some understanding of using and Reference:, et al.

**Comment by K4TFJ** [comment]
@RFC1920  have you been successful in writing a plugin via Harmony for Rust? Would love to talk.. Catch me on Discord (see my profile)

**Comment by RFC1920** [comment]
So far only within the confines of this pile of code.

**Comment by RFC1920** [comment]
With some adjustment I can force the compiler to load other DLLs, and they appear to load from the Managed folder, which is nice.  But, they don't get referenced automatically, so will have to play with the syntax tree "Usings" I suppose.
